.github/workflows/deploy-experimental.yml

```diff
@@ -32,9 +32,9 @@ jobs:
           state: open
       # should build first and save the artifact
       - name: Setup Node
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
-          node-version: '14.x'
+          node-version: '15.x'
```

Comment:
we should only use even-numbered releases of node -- odd-numbered versions are not promoted to LTS.  See here: https://github.com/nodejs/release#release-schedule

So we should set it to 16, not 15.

Reply:
We should probably update that in our README to be 16+ as well.